packages/beacon-node/src/network/reqresp/handlers/blobSidecarsByRange.ts

```diff
@@ -103,8 +105,12 @@ export function validateBlobSidecarsByRangeRequest(
     throw new ResponseError(RespStatus.INVALID_REQUEST, "startSlot < genesis");
   }
 
-  if (count > MAX_REQUEST_BLOCKS_DENEB) {
-    count = MAX_REQUEST_BLOCKS_DENEB;
+  const maxRequestBlobSidecars = isForkPostElectra(config.getForkName(startSlot))
```

Comment:
could consider to add a helper function for this similar to
https://github.com/ChainSafe/lodestar/blob/bb0a4c4f9ac81c54c6a36e3024c75d416b97f07c/packages/config/src/forkConfig/types.ts#L42-L43

Reply:
added helper